src/analysis/retail/evoker/preservation/modules/features/EssenceDetails.tsx

```diff
@@ -5,10 +5,7 @@ import Statistic from 'parser/ui/Statistic';
 import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 import ResourceBreakdown from 'parser/shared/modules/resources/resourcetracker/ResourceBreakdown';
 import { Panel } from 'interface';
-
-import { defineMessage } from '@lingui/macro';
-import { ThresholdStyle, When } from 'parser/core/ParseResults';
-import { TALENTS_EVOKER } from 'common/TALENTS';
+import { ThresholdStyle } from 'parser/core/ParseResults';
```

Comment:
Unless I'm missing something I don't think you need this import or suggestionThresholds anymore with the checklist gone